### ISSUE TITLE
Item display map pin icons are 404, showing alt text

### ISSUE THREAD

**Issue by mberkowski** [Minitex/mdl_search#184]
The map at the bottom of item view screens is missing the png for marker pins, and instead showing alt text for the broken images. There appears to be some extra junk at the end of the image asset URL:

```
https://collection.mndigital.org/assets/marker-icon-574c3a5cca85f4114085b6841596d62f00d7c892c7b03f28cbfa301deb1dc437.png%22)marker-icon.png
``` 
note `%22)marker-icon.png` at the end, all superfluous instead of the correct URL
```
https://collection.mndigital.org/assets/marker-icon-574c3a5cca85f4114085b6841596d62f00d7c892c7b03f28cbfa301deb1dc437.png
```
